tests/test_flatten.py

```diff
@@ -31,3 +44,31 @@ def test_flatten_dict():
             "contextInfo_context2": "info2",
         }
     ]
+
+
+def test_flatten_pd():
+    flattened_pd = flatten(
+        PD_DATA, nested_columns=["contextInfo", "neighborhood"]
+    )
+    assert flattened_pd.equals(
+        DataFrame(
+            [
+                {
+                    "answer": 42,
+                    "contextInfo_context1": "info1",
+                    "contextInfo_context2": "info2",
+                }
+            ]
+        )
+    )
+
+
+def test_aux_function_is_called():
+    mock_flatten_df = Mock()
+
+    with pytest.MonkeyPatch().context() as m:
+        m.setattr("crossfire._flatten_df", mock_flatten_df)
+
+        flatten(PD_DATA, nested_columns=["contextInfo"])
+
+    mock_flatten_df.assert_called_once()
```

Comment:
Is there a reason to use the unnecessarily complicated monkey patch from `pytest` instead of the native `unittest.mock` used [in other tests in this project](https://github.com/search?q=repo%3AFelipeSBarros%2Fcrossfire%20unittest.mock&type=code)?